webdriver/tests/bidi/session/subscribe/invalid.py

```diff
@@ -130,3 +130,45 @@ async def test_subscribe_to_closed_tab(bidi_session):
     # Try to subscribe to the closed context
     with pytest.raises(NoSuchFrameException):
         await bidi_session.session.subscribe(events=["log.entryAdded"], contexts=[new_tab["context"]])
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("value", [True, "foo", 42, {}])
+async def test_params_userContexts_invalid_type(bidi_session, value):
```

Comment:
nit: userContext -> user_context, we usually stick to snake case for test function names